### PR TITLE
bugfix - FDREvent not converting TargetFileName to event.TargetFileName

### PR DESCRIPTION
**Pull request by akozlovets098**
fix for the issue https://github.com/panther-labs/pySigma-backend-panther/issues/62

### CODE REVIEW

File: tests/test_crowdstrike_panther_pipeline.py [hunk 1]
```diff
@@ -78,10 +79,60 @@ def test_basic(mock_click):
...
+        """
+    )
+
+    expected = """def rule(event):
+    if all(
+        [
+            event.deep_get("event_platform", default="") == "Windows",
+            event.deep_get("event_simpleName", default="")
+            in ["ProcessRollup2", "SyntheticProcessRollup2"],
+            event.deep_get("ParentBaseFileName", default="") == "MonitoringHost.exe",
```
Comment (arielkr256) [inline tests/test_crowdstrike_panther_pipeline.py:128]: This should be `event.deep_get("event", "ParentBaseFileName"...` based on the field mapping doc https://docs.google.com/spreadsheets/d/1APjsehfUUSxULuFQYkZYDbVfjyss_tbz0-ZfZOmHtF8/edit?gid=0#gid=0

Comment (arielkr256) [inline tests/test_crowdstrike_panther_pipeline.py:128]:
```suggestion
            event.deep_get("event", "ParentBaseFileName", default="") == "MonitoringHost.exe",
```

File: tests/test_crowdstrike_panther_pipeline.py [hunk 2]
```diff
@@ -78,10 +79,60 @@ def test_basic(mock_click):
                             "KeyPath": "ParentBaseFileName",
```
Comment (arielkr256) [inline tests/test_crowdstrike_panther_pipeline.py:79]: This keypath should be `"event.ParentBaseFileName"` as well.

Comment (akozlovets098) [reply tests/test_crowdstrike_panther_pipeline.py:79]: Should it? We have separate [transformation for process creation](https://github.com/panther-labs/pySigma-backend-panther/blob/main/sigma/pipelines/panther/crowdstrike_panther_pipeline.py#L47-L58) events and its result doesn't include `event`

Comment (arielkr256) [reply tests/test_crowdstrike_panther_pipeline.py:79]: Yes, according to the mapping doc they both should start with `event.`

**Pull request merged**